File: 1960-check-if-the-sentence-is-pangram/check-if-the-sentence-is-pangram.kt
	class Solution {
		fun checkIfPangram(sentence: String): Boolean {
			val alphabet = "abcdefghijklmnopqrstuvwxyz"
			for (c in alphabet) {
				if (!sentence.contains(c)) {
					return false
				}
			}
			return true
		}
	}

File: 1960-check-if-the-sentence-is-pangram/check-if-the-sentence-is-pangram.swift
class Solution {
    func checkIfPangram(_ sentence: String) -> Bool {
        var tempArr = [Int]()
        for i in 0 ..< 26 {
            tempArr.append(i + 97)
        }
        var tempSet = Set([Int]())
        for i in sentence {
            let tempAscii = Int(i.asciiValue!)
            tempSet.insert(tempAscii)
        }

        return Set(tempArr) == tempSet
    }
}